src/purchase_listinator/adapters/in/shopping_cart_event.clj

```diff
@@ -0,0 +1,13 @@
+(ns purchase-listinator.adapters.in.shopping-cart-event
+  (:require [schema.core :as s]
+            [purchase-listinator.models.internal.shopping-cart :as models.internal.shopping-cart]
+            [purchase-listinator.wires.in.shopping-cart :as wires.in.shopping-cart]
+            [purchase-listinator.adapters.misc :as adapters.misc]))
+
+(s/defn wire->internal :- models.internal.shopping-cart/OrderCategoryEvent
```

Comment:
Shouldn't be a fixed event (OrderCategoryEvent), this function should return the CartEvent